src/app/logout/page.tsx

```diff
@@ -1,19 +1,16 @@
 'use client'
-//import { useW3 } from "@w3ui/react"
+import { useW3 } from "@w3ui/react"
 import { useRouter } from "next/navigation"
 import { useEffect } from "react"
 
 export default function LogoutPage () {
-  //const [, { unloadAgent }] = useW3()
+  const [, { logout }] = useW3()
```

Comment:
implementing logout in the client would probably help simplify this method:

https://github.com/web3-storage/w3ui/blob/main/packages/react/src/providers/Provider.tsx#L76

but I think we'd still need something at this level because we need to clear the client, space, and accounts state

Reply:
we _could_ probably use the same event trick to clear space and accounts, but afaict `Client` isn't really meant to be mutable, and the creating a new private key is something that happens during `Client` creation, so we really do need something that can throw away the old client and create a new one, which I don't think can happen inside a client method